include/server/game/constants.hpp

```diff
@@ -17,9 +17,14 @@
 #define TIMESTEP_LEN			std::chrono::milliseconds(30)
 #define	MAX_PLAYERS				4
 
-//	SmartVector capacities
-#define	MAX_NUM_OBJECTS			100
-#define MAX_NUM_BASE_OBJECTS	100
+/*	Maze Constants */
+#define	MAZE_DIRECTORY			"maps"
+#define DEFAULT_MAZE_FILE		"tyler_test.maze"
```

Comment:
This could be nice to move into our `config.json` at the root of the repo with the other server setup stuff so that we don't have to recompile the server when the map changes.

Reply:
Works great on my end! Able to change maps without recompiling. Thanks for adding this feature!